lib/options.json

```diff
@@ -292,11 +292,15 @@
         }
       ]
     },
-    "profile": {
-      "type": "boolean"
-    },
     "progress": {
```

Comment:
Let's move `progress` to `client` object, because this plugin only for `client`

Reply:
It seems it's also used for the CLI output: https://github.com/webpack/webpack-dev-server/blob/ccdce3585ade018211646097cb753c8eb49bf697/lib/Server.js#L86-L89

Reply:
> Aren't progress and profile for the webpack compilation?

For use no, we just emit them to client using `sockets`

Reply:
oh, we should remove this, this plugin is unnecessary, `webpack serve --profile` should enable this plugin on `webpack-cli` side, I think it is already works like this, but we need to test it, adding progress plugin is out of scope webpack-dev-server, also if somebody already use this plugin in this configuration we will adding plugin twice

Reply:
Should we add `clientProgress` or `consoleProgress`, or would it be too much?

Reply:
Can we add tests for cli too to ensure `--progress` works fine via CLI for node (for browser I think we already have tests)

Reply:
Like `webpack serve --progress` and look at output

Reply:
I cannot find a way to mock `stderr` with the TTY support. Wrapping `process.stderr` and passing `stderr: 'inherit'` to `execa` doesn't work.

Reply:
Why we need mock here? Why just do not run `execa` and get `stderr`?

Reply:
I thought it was TTY issue that it didn't print the progress. However, it seems webpack-cli doesn't add `WebpackCLIPlugin` when `webpack serve` is called.